scripts/travis/before_build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-yarn global add grunt-cli
+yarn global add grunt-cli@1.2.0
```

Comment:
Trying this locally, it seems that it will install the specific version (i.e. `--exact` makes no difference).
Probably a docs issue.